src/components/font_size_editor/font_size_editor.ts

```diff
@@ -110,6 +113,13 @@ export class FontSizeEditor extends Component<Props, SpreadsheetChildEnv> {
       this.props.onToggle();
     }
   }
+
+  onInputWheel(ev: WheelEvent) {
```

Comment:
That doesn't really fix the issue does it ? We can still have the same bug (onchange triggering when clicking on a menu item) if we input a value with the keyboard, don't confirm the input, then click a menu item.

Maybe we should go the other direction, and make the menu item change the input value ? (idk if the order of events allow this tho)

Reply:
Input `onChange` is triggered before the `onClick` of the dropdown menu item. As a result, when the user changes the input value and then clicks a menu item, the onChange sets the font size and closes the dropdown before the onClick can be processed, leading to the click being ignored.

Can we use maybe a 100ms debounce on the `setSize`. To ensure that if consecutive calls to `onChange` and `onClick` occurs, only the last one is applied?

Reply:
Debounce sounds like a good idea that would solve the problem :)